src/Avalonia.X11/X11Screens.cs

```diff
@@ -82,6 +82,39 @@ private void OnEvent(XEvent ev)
                     _cache = null;
             }
 
+            private unsafe Size? GetPhysicalMonitorSizeFromEDID(IntPtr rrOutput)
+            {
+                if(rrOutput == IntPtr.Zero)
+                    return null;
+                var output = Marshal.ReadInt64(rrOutput);
+                var properties = XRRListOutputProperties(_x11.Display,output, out int propertyCount);
+                var hasEDID = false;
+                for(var pc = 0; pc < propertyCount; pc++)
+                {
+                    if(properties[pc] == _x11.Atoms.RR_PROPERTY_RANDR_EDID)
+                        hasEDID = true;
+                }
+                if(!hasEDID)
+                    return null;
+                XRRGetOutputProperty(_x11.Display, output, _x11.Atoms.RR_PROPERTY_RANDR_EDID, 0, 128, false, false, _x11.Atoms.AnyPropertyType, out int actualType, out int actualFormat, out int nItems, out _, out IntPtr prop);
+                if(actualType != 19) // XA_INTEGER
+                    return null;
+                if(actualFormat != 8)
+                    return null;
+
+                var edid = new byte[nItems];
+                Marshal.Copy(prop,edid,0,nItems);
+                XFree(prop);
+                XFree(new IntPtr(properties));
+                if(edid.Length < 22)
+                    return null;
+                var width = edid[21];
```

Comment:
Since those are magic numbers, please, quote the standard, e. g.
```cs
// 15h 1 Max. Horizontal Image Size cm.
// 16h 1 Max. Vertical Image Size cm.
```